cg/store/crud/read.py

```diff
@@ -1481,3 +1481,10 @@ def get_illumina_flow_cell_by_internal_id(self, internal_id: str) -> IlluminaFlo
             flow_cells=self._get_query(table=IlluminaFlowCell),
             internal_id=internal_id,
         ).first()
+
+    def get_cases_for_sequencing_qc(self) -> list[Case]:
+        """Return all cases that are ready for sequencing QC."""
+        return apply_case_filter(
+            cases=self._get_query(table=Case),
+            filter_functions=[CaseFilter.PENDING_OR_FAILED_SEQUENCING_QC, CaseFilter.HAS_SAMPLES],
+        ).all()
```

Comment:
Should there not be some condition of it having been run? Or does the "HAS_SAMPLES" do that?

Reply:
Because currently everything that has not finished running would be set to FAILED right?

Reply:
Hm, what do you mean by having been run? The sequencing QC is done on cases after sequencing and before analysis. So I don't think it is necessary to do any additional filtering

Reply:
Yeah, but I mean now we are picking up all cases with pending sequencing qc. Is that not the default that all completely new cases have?

Reply:
So freshly ordered cases will immediately be picked up by this? Or am I mistaken?

Reply:
Ah, true 👍 What do you think is a good approach to avoid that?

Reply:
Lets use the has sequence filter 👍